content/blog/posts/2022-05-19-five-tips-to-improve-your-terminal-experience.md

```diff
@@ -0,0 +1,70 @@
+---
+authorId: rwwagner90
+categories:
+  - devtools
+date: '2022-05-19'
+slug: five-tips-to-improve-your-terminal-experience
+title: 5 Tips to Improve Your Terminal Experience
+---
+
+We've come a long way from the stereotypical terminals seen in the movies with
+green text on black backgrounds and developers furiously
+slinging commands from memory - all on an eight inch screen.
+
+There are now lots of great options for improving both the style and functionality
+of your terminal, to provide you with the best development experience possible.
+
+# Fish
+
+[![ASCII art depicting a fish](/img/blog/five-tips-to-improve-your-terminal-experience/fish.png)](https://fishshell.com/)
+
+One of the biggest improvements to my experience in the terminal was from switching from `bash` to
+[fish](https://fishshell.com/). Fish upgrades your terminal with features like autosuggestions, that
+remember commands you have run before and try to auto complete them for you, and automatically
+generates auto completes for other utilities by parsing their man pages, neat!
+
+# Starship
+
+<a href="https://starship.rs/">
+  <video muted="muted" autoplay="autoplay" loop="loop" playsinline="" class="demo-video">
+    <source src="/video/demo.webm" type="video/webm"> 
+    <source src="/video/demo.mp4" type="video/mp4">
+  </video>
+</a>
+
+<br/>
+
+[Starship](https://starship.rs/) is a cross-shell prompt that is blazing-fast, and infinitely customizable.
+It is built in Rust, which gives it best-in-class speed and safety, to make your prompt as quick and reliable
+as possible. Out of the box it shows you things like what git branch you are on, what version of node, Rust, etc
```

Comment:
"Out of the box, it shows you pertinent information like what git branch you are on, what version of [tool? language] (e.g. node, Rust, etc)
you are using, how long commands took to run, and so much more!